tethysext/atcore/tests/utilities/sqlalchemy_helpers.py

```diff
@@ -39,8 +39,7 @@ def setup_transaction_and_session(self):
         self.session = Session(connection)
 
     def tear_down_session_and_transaction(self):
-        self.session.close()
-        self.transaction.rollback()
+        self.session.close()  # Nested transactions seem to be deassociated on commit on 1.4
```

Comment:
Was this comment meant to read "on close" rather then "on commit"?

Reply:
See https://github.com/sqlalchemy/sqlalchemy/issues/6155

We could add the "if is_active" check before we try to rollback but for now I'd be okay to just add it back since the tests need further revision anyway.

Reply:
So the reason for this change is that you were getting lots of warnings when running tests?

Reply:
Yes, lots of them.